namui/src/namui/system/typeface/load_sans_typeface_of_all_languages.rs

```diff
@@ -34,9 +42,15 @@ pub async fn load_sans_typeface_of_all_languages() -> Result<(), Box<dyn std::er
     let typeface_file_urls: TypefaceFileUrls = get_typeface_file_urls().await?;
 
     let typeface_files = get_typeface_files(&typeface_file_urls).await?;
-    typeface_files
-        .iter()
-        .for_each(|(typeface_type, bytes)| crate::typeface::load_typeface(&typeface_type, bytes));
+    typeface_files.iter().for_each(|(typeface_type, bytes)| {
+        crate::typeface::load_typeface(&typeface_type, bytes);
+
+        let Some(typeface) = crate::typeface::get_typeface(typeface_type.clone()) else {
+            log!("Could not load typeface {:?}", typeface_type);
+            return;
+        };
+        load_default_font_of_typeface(typeface);
+    });
```

Comment:
I think it's better to return typeface on `load_typeface` and use it to load default font, because you handle the case which should not happen with if-else.

Reply:
Good idea. thx

Reply:
I think it's not that much good idea. I think it's better to make arc or typeface and use it for loading typeface and default font.

Reply:
Getting simpler.